client/src/components/Upload/DragAndDropModal.vue

```diff
@@ -26,24 +28,35 @@ function onDrop(event) {
         });
     }
 }
+
+function onDropFail(event) {
+    if (event.dataTransfer?.files?.length > 0) {
+        Toast.error("Please try again", "File failed");
```

Comment:
The file drop did not fail, it was cancelled. This makes it seem like there was an uncaught error, rather than expected behaviour. Same goes for the naming of the new handler in the composable.

Reply:
Change made.